dom/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>DOM</title>
+
+    <!--Link of external css-->
+    <link rel="stylesheet" href="assets/css/reset.css" />
+    <link rel="stylesheet" href="assets/css/styles.css" />
+  </head>
+  <body>
+    <!--Start of wrapper-->
+    <div class="wrapper">
+      <!--Start of header-->
+      <header class="header">
+        <h3>FRIENDS</h3>
+      </header>
+      <!--End of header-->
+      <!--Start of container-->
+      <div class="container" id="container"></div>
```

Comment:
It would be good to have intuitive id names, like contacts-container instead of just container.